### ISSUE TITLE
CHANGELOG.md for 1.11.596 has wrong issue link

### ISSUE THREAD

**Issue by larrywest** [aws/aws-sdk-java#2056]
In the [CHANGELOG for 1.11.596](https://github.com/aws/aws-sdk-java/blob/master/CHANGELOG.md#111596-2019-07-22), the BugFix issue is #1919, but the link is both somewhat garbled and incorrectly references the `aws-sdk-java-v2` project.

The current _displayed_ text (2nd sentence of BugFix issue):
```
See [aws/aws-sdk-java[#1919](https://github.com/aws/aws-sdk-java-v2/issues/1919)](https://github.com/aws/aws-sdk-java/issues/1919) for more information.
```

The proper _MarkDown_ text of the final sentence could be either:
```
See aws/aws-sdk-java #1919 for more information.
```
Or (also _MarkDown_):
```
See [aws-sdk-java #1919](https://github.com/aws/aws-sdk-java/issues/1919) for more information.
```

I will submit PR shortly.

**Comment by debora-ito** [comment]
Thank you for the contribution. The PR is pending review.

**Comment by debora-ito** [comment]
Fixed by #2057.
